numba/cuda/tests/cudapy/test_math.py

```diff
@@ -497,6 +507,13 @@ def test_math_log(self):
         self.unary_template_int64(math_log, np.log, start=1)
         self.unary_template_uint64(math_log, np.log, start=1)
 
+    #---------------------------------------------------------------------------
+    # test_math_log2
+
+    def test_math_log2(self):
+        self.unary_template_float32(math_log2, np.log2, start=1)
+        self.unary_template_float64(math_log2, np.log2, start=1)
```

Comment:
You should probably also test with integer arguments, similarly to `test_math_log`:

```suggestion
        self.unary_template_float64(math_log2, np.log2, start=1)
        self.unary_template_int64(math_log2, np.log2, start=1)
        self.unary_template_uint64(math_log2, np.log2, start=1)
```